utilities/test_suite/HIP/testAllScript.sh

```diff
@@ -0,0 +1,292 @@
+#!/bin/bash
+
+# <<<<<<<<<<<<<< DEFAULT SOURCE AND DESTINATION FOLDERS (NEED NOT CHANGE) >>>>>>>>>>>>>>
+
+cwd=$(pwd)
+
+#Input Images - Three images (mixed size)
+DEFAULT_SRC_FOLDER_1="$cwd/../TEST_IMAGES/three_images_mixed_src1"
+DEFAULT_SRC_FOLDER_2="$cwd/../TEST_IMAGES/three_images_mixed_src2"
+
+# Input Images - Three images (224 x 224)
```

Comment:
L11-L25 all refer to rpp-unittests folder. Lets not refer to that folder if we plan to remove it in the future.

Reply:
Removed them